FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Echo_Qemu_mps2/main_networking.c

```diff
@@ -180,7 +180,7 @@ void main_tcp_echo_client_tasks( void )
         }
         #endif /* ( ipconfigUSE_DHCP != 0 ) */
 
-        memcpy( ipLOCAL_MAC_ADDRESS, ucMACAddress, sizeof( ucMACAddress ) );
+        //memcpy( ipLOCAL_MAC_ADDRESS, ucMACAddress, sizeof( ucMACAddress ) );
```

Comment:
Was this commented out on purpose?

Reply:
Thanks a lot for this feedback. Instead of commenting out ipLOCAL_MAC_ADDRESS, I've now
removed it completely from the source code. (Just as done within other Demo source code.)

I've also fixed formatting errors, so hopefully this resolves now all open requests. Let me know
if anything else needs attention.

best regards,

Florian La Roche